### PR TITLE
add tabbed content to instrumentation in js

### PR DESCRIPTION
**Pull request by pascal-sochacki**
Hey, I added some tabbed content for the instrumentation in javascript. I hope I did everything accordingly. There are still some snippets left to do. I will try to get them done next week. :)

### CODE REVIEW

File: content/en/docs/instrumentation/js/instrumentation.md [hunk 1]
```diff
@@ -33,10 +33,45 @@ npm install \
...
+
+{{< tab TypeScript >}}
+/*tracing.ts*/
+import {BatchSpanProcessor, ConsoleSpanExporter} from "@opentelemetry/sdk-trace-base";
+import {Resource} from "@opentelemetry/resources";
+import {SemanticResourceAttributes} from "@opentelemetry/semantic-conventions";
+import {NodeTracerProvider} from "@opentelemetry/sdk-trace-node";
+import {registerInstrumentations} from "@opentelemetry/instrumentation";
+
+
```
Comment (cartermp) [inline content/en/docs/instrumentation/js/instrumentation.md:49]:
```suggestion
import { BatchSpanProcessor, ConsoleSpanExporter } from "@opentelemetry/sdk-trace-base";
import { Resource } from "@opentelemetry/resources";
import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
import { registerInstrumentations } from "@opentelemetry/instrumentation";

```

File: content/en/docs/instrumentation/js/instrumentation.md [hunk 2]
```diff
@@ -87,10 +132,42 @@ npm install \
...
 
-```javascript
+{{< tabpane langEqualsHeader=true >}}
+{{< tab TypeScript >}}
+import {Resource} from "@opentelemetry/resources";
+import {SemanticResourceAttributes} from "@opentelemetry/semantic-conventions";
+import {WebTracerProvider} from "@opentelemetry/sdk-trace-web";
+import {registerInstrumentations} from "@opentelemetry/instrumentation";
+import {BatchSpanProcessor, ConsoleSpanExporter} from "@opentelemetry/sdk-trace-base";
+
```
Comment (cartermp) [inline content/en/docs/instrumentation/js/instrumentation.md:145]:
```suggestion
import { Resource } from "@opentelemetry/resources";
import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
import { WebTracerProvider } from "@opentelemetry/sdk-trace-web";
import { registerInstrumentations } from "@opentelemetry/instrumentation";
import { BatchSpanProcessor, ConsoleSpanExporter } from "@opentelemetry/sdk-trace-base";
```

**Review by cartermp — APPROVED** [review]
Overall looks good! I think the main question I have is if you'd like to add some type annotations or not. I know the compiler can infer the types correctly for `const`-bound values and it'll work just fine, and that's my personal preference as well. But I'm not sure if it's common to introduce a type annotation for demonstrative purposes or not.

**Comment by pascal-sochacki** [comment]
On the previous [page](https://opentelemetry.io/docs/instrumentation/js/libraries/#node-autoinstrumentation-package), there are no type annotations as well. I would keep it that way

**Review by svrnm — APPROVED** [review]
lgtm

**Comment by svrnm** [comment]
@open-telemetry/javascript-approvers please review/approve.

**Review by legendecas — APPROVED** [review]

**Pull request merged**